libs/core/src/main/java/org/opensearch/action/package-info.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/** Core action module */
+package org.opensearch.action;
```

Comment:
```suggestion
package org.opensearch.core.action;
```
?

Reply:
Or you would like to minimize the change for now? (not sure at the moment what is the plan do, https://github.com/opensearch-project/OpenSearch/issues/8110 would need this change anyway)

Reply:
I opened this PR to just move the classes to the core libraries w/o changing the namespace. After it is merged I'll rebase #7783 which will be the PR to refactor to the `core` namespace.